assignment.js

```diff
@@ -0,0 +1,31 @@
+let numberClicked = false;
+function add(char) {
+    if(numberClicked == false) { // 이전에 연산자 입력,
+        if(isNaN(char)==true) { // 입력값이 또 연산자이면,
+            // 아무 일도 일어나지 않았다.
+        } else { // 연산자가 아닐 시
+            document.getElementById('display').value += char;
+        }
+    } else { // 이전에 숫자 입력시 
+        document.getElementById('display').value += char;
+    }
+
+    if(isNaN(char)==true) {
```

Comment:
```suggestion
    if(Number.isNaN(char)==true) {
```

isNaN의 경우 아래 링크에서 `Number.isNaN`으로 바꾸는 것을 권장하고 있어요!

확인해 보시면 좋을 듯 합니다 ~
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/isNaN

Reply:
확인했습니다!! number.을 붙여주지 않으면 인자를 강제로 Number로 변환하는군요,,